docs/development.md

```diff
@@ -233,3 +233,61 @@ This is GitHub user that does all the automation.
 | Script      | What it is doing                          | Access via                                    |
 |-------------|-------------------------------------------|-----------------------------------------------|
 | .travis.yml | Uploading binaries to GitHub release page | Personal access token `deploy-github-release` |
+
+
+## Licenses
+
+A software license is a legal instrument governing the use or redistribution of software.
+License compatibility is a legal framework that allows for pieces of software with different
+software licenses to be distributed together. The need for such a framework arises because
+the different licenses can contain contradictory requirements, rendering it impossible to
+legally combine source code from separately-licensed software in order to create and
+publish a new program.
+
+Odo is Apache v2 licensed and we vendor a number of different packages for various
+reasons which adds the responsibility of being mindful of the compatibility of licenses
+of our vendor-ed packages with that of Odo. So, we decided, to add checks for the
+compatibility of every vendor-ed package of Odo, for which, we use the tool
+[wwhrd](https://github.com/frapposelli/wwhrd). Odo license compatibility checks are added
```

Comment:
Again no need for this...

Reply:
I thought of adding this because of "why you add this" part of https://github.com/redhat-developer/odo/pull/744#issuecomment-422072935. anyway, I'll remove it